components/style/properties.mako.rs

```diff
@@ -72,12 +72,34 @@ class Shorthand(object):
         self.sub_properties = [LONGHANDS_BY_NAME[s] for s in sub_properties]
         self.internal = internal
 
+class Method(object):
+    def __init__(self, name, return_type=None, arg_types=[], is_mut=False):
```

Comment:
While it might not be problematic in this specific scenario, I wouldn't recommend specifying a `list` as a default parameter:

http://docs.python-guide.org/en/latest/writing/gotchas/#mutable-default-arguments